src/classes/Wavetable.tsx

```diff
@@ -0,0 +1,25 @@
+export default class Wavetable {
```

Comment:
I'm cool moving forward with this, but we may want to discuss either standardization around coding paradigms, or totally encapsulate entire back end as class based and the entire UI controller elements as function based. Though that honestly might be how it's already divided.